railgun/test/integration/ingress_https_test.go

```diff
@@ -355,6 +355,38 @@ func TestHTTPSIngress(t *testing.T) {
 		}
 	}()
 
+	t.Logf("checking ingress %s status readiness.", ingress1.Name)
+	assert.Eventually(t, func() bool {
+		curIng, err := env.Cluster().Client().NetworkingV1().Ingresses(corev1.NamespaceDefault).Get(ctx, ingress1.Name, metav1.GetOptions{})
+		if err != nil || curIng == nil {
+			return false
+		}
+		ingresses := curIng.Status.LoadBalancer.Ingress
+		for _, ingress := range ingresses {
+			if len(ingress.Hostname) > 0 || len(ingress.IP) > 0 {
+				t.Logf("networkingv1 ingress1 hostname %s or ip %s is ready to redirect traffic.", ingress.Hostname, ingress.IP)
+				return true
+			}
+		}
+		return false
+	}, 120*time.Second, 1*time.Second, true)
```

Comment:
2 minutes seems like a long time for the status updates to be allowed, should we potentially try something shorter?
```suggestion
	}, 30*time.Second, 1*time.Second, true)
```

Reply:
2 minutes is threshold. would keep as it is.

Reply:
Ok, just wanted to check thanks :+1: